nzedb/Contents.php

```diff
@@ -128,34 +154,54 @@ public function getByID($id, $role)
 		return $this->row2Object($row);
 	}
 
+	/**
+	 * @param $content
+	 *
+	 * @return mixed
+	 */
 	public function validate($content)
 	{
-		if (substr($content->url, 0, 1) != '/') {
-			$content->url = "/" . $content->url;
+		if (substr($content->url, 0, 1) !== '/') {
+			$content->url = '/' . $content->url;
 		}
 
-		if (substr($content->url, strlen($content->url) - 1) != '/') {
-			$content->url = $content->url . "/";
+		if (substr($content->url, strlen($content->url) - 1) !== '/') {
+			$content->url .= '/';
 		}
 
 		return $content;
 	}
 
+	/**
+	 * @param $form
+	 *
+	 * @return false|int|string
+	 */
 	public function add($form)
 	{
 		$content = $this->row2Object($form);
 		$content = $this->validate($content);
-		if ($content->ordinal == 1) {
-			$this->pdo->queryDirect("UPDATE page_contents SET ordinal = ordinal + 1 WHERE ordinal > 0");
+		if ($content->ordinal === 1) {
+			$this->pdo->queryDirect('UPDATE page_contents SET ordinal = ordinal + 1 WHERE ordinal > 0');
 		}
 		return $this->data_add($content);
 	}
 
+	/**
+	 * @param $id
+	 *
+	 * @return bool|\PDOStatement
```

Comment:
Does not look like ->queryExec() can return true so it and this should be updated to be \PDOStatement|false